test/positivity.lean

```diff
@@ -134,6 +135,9 @@ example {X : Type*} [metric_space X] (x y : X) : 0 ≤ dist x y := by positivity
 example {E : Type*} [add_group E] {p : add_group_seminorm E} {x : E} : 0 ≤ p x := by positivity
 example {E : Type*} [group E] {p : group_seminorm E} {x : E} : 0 ≤ p x := by positivity
 
+example (ε : ℝ) (l : ℕ) : 0 < szemeredi_regularity.initial_bound ε l := by positivity
+example (ε : ℝ) (l : ℕ) : 0 < szemeredi_regularity.bound ε l := by positivity
```

Comment:
I suspect we're better off testing these in the regularity/bound file, rather than here; since this extension is essentially designed purely for the regularity lemma